libindy/src/api/payments.rs

```diff
@@ -770,11 +771,11 @@ pub extern fn indy_build_payment_req(command_handle: i32,
 ///   Note: this param will be used to determine payment_method
 ///
 /// #Returns
-/// utxo_json - parsed (payment method and node version agnostic) utxo info as json:
+/// receipts_json - parsed (payment method and node version agnostic) receipts info as json:
 ///   [{
-///      txo: <str>, // UTXO input
-///      paymentAddress: <str>, //payment address for this UTXO
-///      amount: <int>, // amount of tokens in this input
+///      receipt: <str>, // source input
```

Comment:
@Artemkaaas @vimmerru Is comment here correct?

Reply:
receipt: <str>, // receipt that can be used for payment referencing and verification